spyderlib/plugins/configdialog.py

```diff
@@ -90,8 +91,21 @@ def apply_changes(self):
             self.save_to_conf()
             if self.apply_callback is not None:
                 self.apply_callback()
+
+            # Since the language cannot be retrieved by CONF and the language
+            # is needed before loading CONF, this is an extra method needed to
+            # ensure that when changes are applied, they are copied to a
+            # specific file storing the language value. This only applies to
+            # the main section config.
+            if self.CONF_SECTION == u'main':

```

Comment:
The other way would be to make the save_lang method part of one of the ancestors so that EVERY section would alwyas make that save... but since it is only necessary for main...